lib/widgets/forest_park_map.dart

```diff
@@ -73,10 +80,16 @@ class _ForestParkMapState extends ConsumerState<ForestParkMap> with WidgetsBindi
     // final parkTrails = ref.watch(parkTrailsProvider);
     final locationStatus = ref.watch(locationPermissionStatusProvider);
 
-    final followOnLocation = ref.watch(followOnLocationProvider);
-    ref.listen(followOnLocationProvider, (prev, next) {
-      if (next != prev && next != FollowOnLocationUpdate.never) {
-        _followCurrentLocationStreamController.add(null);
+    final followOnLocation = ref.watch(followOnLocationTargetProvider);
```

Comment:
This isn't used anywhere, so should be removed to prevent unnecessary widget rebuilds.